demisto_sdk/tests/test_files/content_slim/Packs/Sample01/README.md

```diff
@@ -43,6 +43,13 @@ Contributions are welcome and appreciated. For more info, visit our [Contributio
 Contributions are welcome and appreciated. For more info, visit our [Contribution Guide](https://xsoar.pan.dev/docs/contributing/contributing).
 ### Pack Contributors:
 
+---
+ - Contributor1
+ - Contributor2
+
+Contributions are welcome and appreciated. For more info, visit our [Contribution Guide](https://xsoar.pan.dev/docs/contributing/contributing).
+### Pack Contributors:
```

Comment:
what's this?

Reply:
removed, no idea how it was added.. probably some test that didn't finish executing